File: C#L5/Letters.cs
namespace StringLibrary
{
    public class Letters : StringBase
    {
        public Letters(string val) : base(val) { }

        public override int GetLength() => value.Length;

        public override string RemoveChar()
        {
            return value.Replace("a", "").Replace("A", "");
        }
    }
}

File: C++L5/Letters.cpp
#include "Letters.h"

size_t Letters::getLength() const {
    return value.length();
}

std::string Letters::removeChar() {
    std::string result;
    for (char ch : value) {
        if (ch != 'a' && ch != 'A') result += ch;
    }
    return result;
}
